test/integration/controller/job/job_controller_test.go

```diff
@@ -65,14 +60,14 @@ var _ = ginkgo.Describe("Job controller", func() {
 				return false
 			}
 			return createdJob.Spec.Suspend != nil && *createdJob.Spec.Suspend
-		}, timeout, interval).Should(gomega.BeTrue())
+		}, framework.Timeout, framework.Interval).Should(gomega.BeTrue())
```

Comment:
As a follow up, this should be:

```golang
gomega.Eventually(func() *bool {
	if err := k8sClient.Get(ctx, lookupKey, createdJob); err != nil {
		return false
	}
	return createdJob.Spec.Suspend
}, framework.Timeout, framework.Interval).Should(gomega.Equal(pointer.Bool(true))
```

And similarly for the rest of the `Eventually` things

Reply:
will look a little strange sometimes returning true and sometimes false on the Get error since sometimes we check that the job is suspended and sometimes not.

Reply:
Oops, missed that. I guess you want to make sure we retry in the case the apiserver is down. But I don't think that's necessary when the client uses the informer's cache.

So if there's an error, we should just fail the test (but we don't really expect that to happen).

Reply:
ok, lets do that as a followup.